tests/components/fritzbox/__init__.py

```diff
@@ -110,9 +109,7 @@ class FritzDeviceClimateMock(FritzEntityBaseMock):
     target_temperature = 19.5
     window_open = "fake_window"
     nextchange_temperature = 22.0
-    nextchange_endperiod = 0
-    nextchange_preset = PRESET_COMFORT
-    scheduled_preset = PRESET_ECO
```

Comment:
these two properties do not exists on the original FritzDevice, therefore they don't need to be mocked